.github/labeler.yml

```diff
@@ -0,0 +1,56 @@
+sig/engine:
+  - "cmd/*"
+  - "components/backup/*"
+  - "components/collections/*"
+  - "components/engine_*/*"
+  - "components/external_storage/*"
+  - "components/file_system/*"
+  - "components/keys/*"
+  - "components/raft_log_engine/*"
+  - "components/raftstore/*"
+  - "components/rusoto_util/*"
+  - "components/sst_importer/*"
+  - "components/test_raftstore/*"
+  - "components/test_sst_importer/*"
+  - "components/tikv_util/*"
+  - "etc/*"
+  - "metrics/*"
+  - "src/import/*"
+  - "src/server/*"
+  - "src/config.rs"
+  - "src/lib.rs"
+  - "src/read_pool.rs"
```

Comment:
The following does not belong to sig/engine:
```
cmd/*
components/backup/*
components/collections/*
components/external_storage/*
components/keys/*
components/raftstore/*
components/sst_importer/*
components/test_raftstore/*
components/test_sst_importer/*
components/tikv_util/*
etc/*
metrics/*
src/import/*
src/server/*
src/config.rs
src/lib.rs
src/read_pool.rs
```

Reply:
@hi-rustin some of these directories don't related to a particular SIG. How can we handle them?

Reply:
If it involves more than one sig we can add more than one sig labels, or if it is absolutely impossible to divide the sig, we can remove it.